index.html

```diff
@@ -0,0 +1,145 @@
+          <li><a class="side-link" href="about.html">About</a></li>
+          <li><a class="side-link" href="#aboutme">Program</a></li>
+          <li><a class="side-link" href="#contact">Join</a></li>
+          <li><a class="side-link" href="#contact">Sponsor</a></li>
+          <li><a class="side-link" href="#contact">Campaign</a></li>
+        </ul>
+      </div>
+      <div class="text">
+        <p class="greeting-text primary-color">"Hi Music Enthusiats"</p>
+        <h2>Franky<span> Global</span></h2>
+        <h3>Musical Concert <small>2022</small></h3>
+      </div>
+      <div class="sub-text">
+        <p class="hero-text">
+          A joyfyl celebration believing in the value of opennes and
+          sharing, creating a positive change with people from over 
+          80 countries is taking place in October, in Australia.
+        </p>
+        <p class="hero-time">
+          2015.10.15(THU) ~ 16(FRI)
+        </p>
+        <p class="hero-place">
+          @ Upper House Sydney Australia
+        </p>
+      </div>
+    </header>
+  </div>
+  <main>
+    <section class="main-program">
+      <h4 class="program-text">Main Program</h4>
+      <hr class="line">
+      <div class="program-details d-flex">
+        <div class="events">
+          <ul class="d-flex u-list">
+            <li><i class='bx bxs-microphone bx-sm primary-color'></i></li>
+            <li class="criteria primary-color">Interview</li>
+            <li class="exp">
+              Get to interview you favorite band members. 
+              Ask your hear burning questions.Get autographs to showoff with friends
+            </li>
+          </ul>
+          <ul class="d-flex u-list">
+            <li><i class='bx bxs-chalkboard bx-sm primary-color'></i></li>
+            <li class="criteria primary-color">Workshop</li>
+            <li class="exp">
+              Ever dreamt of breaking into the music industry, 
+              well this is your chance to take part in this amazing Workshop
+              held by professionals in the field
+            </li>
+          </ul>
+          <ul class="d-flex u-list">
+            <li><i class='bx bxs-music bx-sm primary-color'></i></li>
+            <li class="criteria primary-color">Rock Session</li>
+            <li class="exp">Get to see your favorite band members 
+              perform on stage, and so much more.
+            </li>
+          </ul>
+          <div class="d-flex u-btn">
+            <button type="button" class="exp-btn program-btn">Join</button>
+          </div>
+        </div>
+        <div class="more">
+          <a href="#" class="see-more tertiary-color">SEE THE WHOLE PROGRAM</a>
+          <hr class="line primary-color">
+        </div>
+      </div>
+      
+    </section>
+    <section class="feature-artists">
+      <h4 class="program-text secondary-color">Team Members</h4>
+      <hr class="primary-color line">
+      <div class="program-details artists">
+        
+      </div>
```

Comment:
- [ ] I highly suggest that you work on this section
 this is the original
![mai](https://user-images.githubusercontent.com/44745855/142427713-06910780-e8a8-4240-bc01-290ceae76211.png)

your page
![org](https://user-images.githubusercontent.com/44745855/142428059-4164f262-e738-4304-8791-e2926c0bfabe.png)

Reply:
Hi @DeliceLydia 
Should I use the same icons?
Should I also add more events?

Reply:
Hello @charlesgobina,
Actually the requirements said that you should replicate the design
only contents you are allowed to personalize them.